remote-codec/remote-codec-server/oidc.go

```diff
@@ -0,0 +1,88 @@
+package main
```

Comment:
Hrmm, can't think of an easy way necessarily, but would be nice if a real Go embedded oauth server could be used and a test existed. But none of that is required for now.